src/components/Modal/NewAppsToAuthorizeModal/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC, useCallback, useState } from "react"
+import { useNavigate } from "react-router-dom"
+import InfoBox from "../../InfoBox"
+import {
+  BodyLg,
+  Button,
+  Card,
+  H5,
+  LabelMd,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  Radio,
+  RadioGroup,
+  Stack,
+} from "@threshold-network/components"
+import { BaseModalProps } from "../../../types"
+import withBaseModal from "../withBaseModal"
+import { useStakingState } from "../../../hooks/useStakingState"
+import { getStakeType } from "../../../utils/getStakeType"
+import { isSameETHAddress } from "../../../web3/utils"
+
+const NewAppsToAuthorizeModal: FC<BaseModalProps> = ({ closeModal }) => {
+  const { stakes } = useStakingState()
+
+  const [selectedAuthorizerAddress, setSelectedAuthorizerAddress] = useState("")
+
+  const navigate = useNavigate()
+
+  const routeToStake = useCallback(() => {
+    if (selectedAuthorizerAddress) {
+      navigate(`/authorize/${selectedAuthorizerAddress}`)
```

Comment:
Is this route correct? Shouldn't it be:

```suggestion
      navigate(`/staking/authorize/${selectedAuthorizerAddress}`)
```